README.md

```diff
@@ -396,17 +396,20 @@ int main(int argc, char* argv[]) {
 
 ## Support
 
-- This project supports Windows, macOS, Linux
-- This project supports C++14 (and higher) compilers (we test with GCC >= 7.3,
-  Clang >= 6.0, and MSVC >= 2017)
-- This project supports Bazel (>= 4.0) and CMake (>= 3.5) builds. See the
-  [Quickstart examples](https://github.com/googleapis/google-cloud-cpp#quickstart)
+- This project follows Google's [Foundational C++ Support Policy][support-policy],
+  which is best understood as the [Foundational C++ Support Matrix][support-matrix].
+  - Language Version (>= C++14)
```

Comment:
Some of these bullet points finish with a period, some do not.  Please make them consistent.